far/farlang.templ.m4

```diff
@@ -21226,32 +21226,32 @@ MViewSearchFor
 "&Rasti"
 
 MViewSearchForText
-"Искать &текст"
-"Search for &text"
-"Hledat &text"
-"Suchen nach &Text"
-"&Szöveg keresése"
-"Szukaj &tekstu"
-"Buscar cadena de &texto"
-"Hľadať &text"
-"Ricerca Di &Testo"
-"Шукати &текст"
-"Шукаць &тэкст"
-"Ieškoti &teksto"
+"&Текст"
+"&Text"
+"&Text"
+"&Text"
+upd:"&Szöveg"
+upd:"Szukaj &tekstu"
```

Comment:
@BestiaPL, please help with Polish translation here.

Reply:
Can you put here screenshot of search dialog (even in English) and tell me how much characters can I use in strings.

Reply:
Thank you. Here you are!

![ViewSearch Pol New](https://user-images.githubusercontent.com/35671029/210045717-22e8df58-1d8e-42a5-8db4-64e60414e320.png)
![ViewSearch Eng New](https://user-images.githubusercontent.com/35671029/210045573-bedfb987-b2f0-4b68-ad30-d9e10f213c7b.png)

If in the future we want to add the "Hex" option to the Search dialog in Editor, we may need to fit all this into 72 characters:
```
Search for  (•) Text  ( ) Hex  [ Word ] [ Selection ]
```

@alabuzhev, what do you think of changing radio button to checkbox here? It'll be easier to fit in one line in some languages.

Reply:
Please, try to set:
&Tekst
S&zesnastkowo

Reply:
Done!

Reply:
> what do you think of changing radio button to checkbox here? It'll be easier to fit in one line in some languages.

Let's solve one thing at a time. Editor does not have a hex search yet so there is nothing to worry about :)

In general, "hex" is probably idiomatic / well known enough to leave it as is?
If not, I suppose it can be shortened in other languages too, same as we don't write "hexadecimal" in English here.

Reply:
ACK